build-tests/api-extractor-lib2-test/config/api-extractor.json

```diff
@@ -4,7 +4,10 @@
   "mainEntryPointFilePath": "<projectFolder>/lib/index.d.ts",
 
   "apiReport": {
-    "enabled": true
+    "enabled": true,
+    "alphaReportFileName": "<unscopedPackageName>.alpha.api.md",
+    "betaReportFileName": "<unscopedPackageName>.beta.api.md",
+    "publicReportFileName": "<unscopedPackageName>.public.api.md"
```

Comment:
>`"description": "The filename for the API report files.  It will be combined with \"reportFolder\" or \"reportTempFolder\" to produce a full file path. The file extension should be \".api.md\", and the string should not contain a path separator such as \"\\\" or \"/\".",`

Design question: Is it really necessary to specify `.alpha.api.md`, `.beta.api.md` in the configuration?  If API Extractor enforces that `reportFilename` MUST end with `.api.md` then an alternative design might be to specify the variations that you want to generate (`alpha`,`beta`,`public`) and then `.api.md` can be transformed to `.alpha.api.md` automatically.  A benefit of this approach is that API Extractor could warn about the presence of deadwood files (e.g. `my-package.beta.api.md` appears in the target folder but is not a configured output).

Reply:
Yeah, if we wanted to enforce that all of these reports end in `api.md` (which doesn't seem unreasonable to me), that would simplify things some. It deviates from the existing pattern for the single report, but the benefits seem worth it. I'm in favor of that if you are.